### DISCUSSION TITLE
Strange faded out sidebar

### DISCUSSION THREAD

**Discussion by danraymond** [NicolasPetton/Notebook#15] [Q&A]
Hi @NicolasPetton , see the attached image

![2021-01-05 13_29_35-Making Fountain Pens — A Collection of How to's — Mozilla Firefox](https://user-images.githubusercontent.com/1908612/103603392-3df43e80-4f5a-11eb-92b2-0f25faaa58a1.png)

Can't work out why the palette seems to fade the sidebar completely. I have installed your theme and reloaded. No matter what palette I choose I get this washed out efect. Any ideas?

**Comment by NicolasPetton** [comment]
Hi @danraymond, 

Could you tell me what version of Notebook you have installed, as well as the list of plugins you have in your wiki?

If you could upload here a wiki (without your data, of course), that would help me reproduce and fix the issue.

**Reply by danraymond** [reply]
Hi @NicolasPetton 

I am using latest notebook theme version from your site.

See attached. No peronal data in the file.
Thank you for offering to assist.

[Pen Making 5123202012.zip](https://github.com/NicolasPetton/Notebook/files/5773327/Pen.Making.5123202012.zip)

**Reply by NicolasPetton** [reply]
Thanks, I found the problem.

The plugin https://github.com/gt6796c/mermaid-tw5 styles all elements with the `.section` css class and sets their opacity to 0.2.
I consider this to be a bug in the plugin, but I can change the CSS class used by Notebook.
